pkg/api/register.go

```diff
@@ -15,7 +15,6 @@ func Setup(ctx context.Context, server *server.Server) error {
 	scaled := config.ScaledWithContext(ctx)
 	image.RegisterSchema(scaled, server)
 	keypair.RegisterSchema(scaled, server)
-	vm.RegisterSchema(scaled, server)
 	vmtemplate.RegisterSchema(scaled, server)
-	return nil
+	return vm.RegisterSchema(scaled, server)
```

Comment:
would prefer to return the error of the method directly and keep return `nil` on the end for better code format.

Reply:
```
func Setup(ctx context.Context, server *server.Server) error {
	scaled := config.ScaledWithContext(ctx)
	image.RegisterSchema(scaled, server)
	keypair.RegisterSchema(scaled, server)
	vmtemplate.RegisterSchema(scaled, server)
	if err := vm.RegisterSchema(scaled, server); err != nil {
		return err
	}
	return nil
}
```

do you mean this?  The last if is redundant.

Reply:
This is not redundant as it clearly states `nil` is the expected of the method and the user doesn't need to check what is returned by the `vm.RegerSchema` method and why it should be put at the end of the method since it looks all the same with other schema registers.